packages/cadl-ranch-specs/http/azure/arm/models/common-types/managed-identity/identity.tsp

```diff
@@ -0,0 +1,99 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+
+namespace Azure.Arm.Models.CommonTypes.ManagedIdentity;
+
+@resource("managedIdentityTrackedResources")
+model ManagedIdentityTrackedResource is TrackedResource<ManagedIdentityTrackedResourceIdentities> {
+  @key("managedIdentityTrackedResourceName")
+  @path
+  @segment("managedIdentityTrackedResources")
+  @doc("arm resource name for path")
+  @pattern("^[A-Za-z0-9]([A-Za-z0-9-_.]{0,62}[A-Za-z0-9])?$")
+  name: string;
+}
+
+@doc("Managed Identity Tracked Resource Identities.")
+model ManagedIdentityTrackedResourceIdentities {
+  @doc("The type of managed identity.")
+  type?: string;
+}
```

Comment:
This is not correct. Please check the usage in the issue:
https://github.com/Azure/cadl-ranch/issues/605

```typescript
model Service is TrackedResource<ServiceProperties> {
  @doc("The name of Azure API Center service.")
  @maxLength(90)
  @minLength(1)
  @pattern("^[a-zA-Z0-9-]{3,90}$")
  @path
  @key("serviceName")
  @segment("services")
  name: string;

  ...ManagedServiceIdentityProperty;
}
```